plugin_tests/client/largeImageSpec.js

```diff
@@ -0,0 +1,68 @@
+/* globals girder, girderTest, describe, it, expect, waitsFor, runs */
+
+girderTest.addCoveredScripts([
+    '/clients/web/static/built/plugins/jobs/plugin.min.js',
+    '/clients/web/static/built/plugins/worker/plugin.min.js',
```

Comment:
These two (`jobs`, `worker`) should probably be in a `girderTest.addScripts`, not `girderTest.addCoveredScripts`.